ui/src/components/facets/FacetsGrid.js

```diff
@@ -10,7 +10,14 @@ function FacetsGrid(props) {
   const updateFacets = props.updateFacets;
   const facetsList = facets.map(facet => (
     <GridListTile key={facet.name}>
-      <FacetCard facet={facet} updateFacets={updateFacets} />
+      <FacetCard
+        facet={facet}
+        selectedValues={getSelectedFacets(
```

Comment:
I have no preference between selectedValues and selectedFacetValues. Let's pick one and use everywhere, in all files.

Reply:
selectedValues is just the list selected facet values in that card. 
selectedFacetValues is a map of facet name to selected values.
They are both different